src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/InputParameterName_1.java

```diff
@@ -0,0 +1,231 @@
+////////////////////////////////////////////////////////////////////////////////
+// Test case file for checkstyle.
+// Created: Feb-2001
+// Ignore violation
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
could you please remove this header comment, do it for all inputs in this PR

Reply:
done